news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Lazer Grand Arena: Registrations now open!"
+date: 2024-06-18 10:00:00 +0000
+---
+
+While it's great to see countries rallying together and forming teams to compete, sometimes we have to wonder what players could truly do if they didn't have to pick around their teammates and truly shine on their own, and with the recent improvements made to osu!'s future client, we feel like it's finally time to bring to you...
+
+![](/wiki/Tournaments/LGA/2024/img/lga2024-banner.jpg)
+
+THE LAZER GRAND ARENA!
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-06-18-lga-announcement.mp4?v=0" type="video/mp4" preload="none">
+    </video>
+</div>
+
+This brand new event will pit the world's 32 best players against eachother in a 1v1 played on **osu!(lazer)!**
+
+The Grand Arena is going to be home to many experiments to the familiar tournament format. Featuring a full featured artist lineup, the players will now battle eachother using **Cumulative Score Difference**! This means that if you win by 20k, you are up by 20k for the rest of the match. After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to do some try out some new exciting ideas for players and viewers alike!
```

Comment:
```suggestion
The Grand Arena is going to be home to many experiments to the familiar tournament format. Featuring a full featured artist lineup, the players will now battle each other using **Cumulative Score Difference**! This means that if you win by 20k, you are up by 20k for the rest of the match. After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to do some try out some new exciting ideas for players and viewers alike!
```

Reply:
applied similar reviews from wala (ty for the quick review though)